e2e/e2e-utils/loadDarksideWallet.js

```diff
@@ -16,6 +16,7 @@ let loadDarksideWallet = async () => {
   await element(by.id("settings.custom-server-chain.regtest")).tap();
```

Comment:
in my testing I needed to scroll `1000` instead `900` to change the server property. The same in `LoadRecipientWallet`...

Reply:
This is the best way to scroll to a specific component:
`await waitFor(element(by.text('Item #5'))).toBeVisible().whileElement(by.id('itemsList')).scroll(50, 'down');`

Reply:
ok ill do that now!